src/handlers/pricing/action.ts

```diff
@@ -46,3 +57,8 @@ export const pricingLabelLogic = async (): Promise<void> => {
     logger.info(`Skipping action...`);
   }
 };
+
+export const isParentIssue = (body: string) => {
+  const parentPattern = /-\s+\[( |x)\]\s+#\d+/;
```

Comment:
This is just checking for a checklist? This should look for the UI element when GitHub shows that it is a "tracking issue" of other issues?

Reply:
Yeah, usually parent issue conatins this string:
`- [ ] #issue_number`.

Reply:
Looks like this achieved the specification. I was concerned about this UI element but perhaps it isn't necessary to implement. 
- https://github.com/ubiquity/ubiquibot/issues/268#issuecomment-1585532148